src/server/templates/register-form/step.njk

```diff
@@ -26,6 +26,24 @@
         {% endif %}
       {% endfor %}
 
+      {% if stepData.keys() == 'email' %}
+        <details>
+          <summary>
+            <span class="details__summary">Why should I provide an email address?</span>
+          </summary>
+
+          <div class="details__content">
+            <div class="callout callout--info">
+              <p>
+                A confirmation will be emailed to you at the end of this service
```

Comment:
is this true? I don't think we want to email a confirmation (question for Trilly really)